Resources/public/css/theme/views/subitemlist.css

```diff
@@ -92,3 +92,69 @@
     left: 50%;
     position: absolute;
 }
+
+.ez-view-subitemlistview .ez-subitemlist-table .ez-subitem-hovered-priority-cell  .ez-subitem-priority-icons:after {
```

Comment:
those selectors seems a bit too complicated.
`.ez-view-subitemlistview .ez-subitem-hovered-priority-cell  .ez-subitem-priority-icons:after` should be enough and easier to read